dora/src/object.rs

```diff
@@ -493,7 +497,7 @@ impl Str {
         let total_len = val.len();
 
         if offset > total_len {
-            return Ref::null();
```

Comment:
Here as well

Reply:
This too: https://github.com/dinfuehr/dora/pull/152/files#diff-edf3ee5de6272371b58c0a2d02e7c112.